test/tests/userobjects/force_preaux/force_preaux.i

```diff
@@ -0,0 +1,70 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1
+  nx = 2
+  ymin = 0
+  ymax = 1
+  ny = 2
+[]
+
+[Variables]
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+    initial_condition = 1
+  [../]
+[]
+
+[Kernels]
+  [./time]
+    type = TimeDerivative
+    variable = u
+  [../]
+
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+[]
+
+
+[Postprocessors]
+  [./total_u]
+    type = ElementIntegralVariablePostprocessor
+    variable = u
+  [../]
+
+  [./scale1]
+    type = ScalePostprocessor
+    value = total_u
+    scaling_factor = 1
+  [../]
+
+  [./scale2]
+    type = ScalePostprocessor
+    value = total_u
+    scaling_factor = 1
+    force_preaux = true
```

Comment:
OK - Good, Just so we start getting better about this can you add a comment either here, or two the top of this new file explaining what you expect to see in this result? Clearly one of these is lagged, so just say that. We need our test suite to be a little more descriptive.